### PR TITLE
1002022 - Check Component ProductDetails

### PR DESCRIPTION
**Pull request by eduardo846**
### Task
Check Component ProductDetails
### Descriptions
"Explanation
When the user clicks the specifications button found on the product-detail page, the user can read the specifications that exist, in this case concerning the Lorem font. When the user clicks it on, the specification text is cleared below the button, which gives more visual content to the product-detail section."
### Images y Gifs
image

### Steps to follow
1.-Compilar

### CODE REVIEW

**Review by e-azocar — CHANGES_REQUESTED** [review]
We are using Tailwind for the app's styles, so I recommend you that use this tool

**Comment by e-azocar** [comment]
It's necessary to follow the design, if you can, please use the design as guide
https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=65%3A2

### Design
image

### Your changes
image

File: README.md [hunk 1]
```diff
@@ -121,8 +121,8 @@ We used `create-react-app` to initialize this project, so it is configurated to
 Now that you are set up, open a terminal and:
 
 ```sh
-git clone https://github.com/Azordev/art-marketplace
-cd art-marketplace
+git clone https://github.com/Azordev/sencico
+cd sencico
```
Comment (Israel-Laguan) [inline README.md:125]: Seems like your branch need a rebase, you have old code

Comment (Israel-Laguan) [reply README.md:125]: @e-azocar can you confirm please?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "axios": "0.24.0",
+    "axios": "^0.24.0",
```
Comment (Israel-Laguan) [inline package.json:6]: I don't see this package being used, please remove if not needed

File: src/components/Products/CategoriesSlider.jsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+/import React from "react";
```
Comment (Israel-Laguan) [inline src/components/Products/CategoriesSlider.jsx:1]: This seems like a typo, please check

File: src/components/Products/Pagination.jsx [hunk 4]
```diff
@@ -1,55 +1,65 @@
...
-              }
-              >Siguiente</a>
-          </li>
-        </ul>
-      </nav>
-    </div>
-  );
-}
+const Pagination = () => {
+	return (
```
Comment (Israel-Laguan) [inline src/components/Products/Pagination.jsx:4]: This is old code, @Eghinner please confirm, seems like a regression.

@eduardo846 I suggest you rebase from origin (downstream on your case)

File: src/pages/Product.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ import { MainNav, Footer } from "../components";
 import { getProductoByIdAsync } from "../actions/productosActions";
 import { isAuthenticated } from "../utils/isAuthenticated";
 import { PUBLIC_URL } from "../config/constants";
+import "../colormodificado.css";
```
Comment (Israel-Laguan) [inline src/pages/Product.js]: Please use English for the name of the file, for example

`Product.module.css`

File: src/pages/Product.js [hunk 6]
```diff
@@ -98,53 +99,53 @@ const Product = () => {
         <div className="flex flex-col w-full px-2 sm:px-8 xl:px-5 2xl:px-11">
           <div className="relative flex flex-col w-full mt-16 lg:flex-row">
             <div className="relative lg:w-1/2 space-y-3 lg:space-y-5 2xl:pr-20">
-              <details className="relative px-6 py-2 overflow-hidden text-black bg-add rounded-lg select-none">
-                <summary className="cursor-pointer">
+              <details className="relative px-6 py-2 overflow-hidden text-black bg-add rounded-lg select-none" id="spancolorfon">
+                <summary className="cursor-pointer" id="spancolor">
                   <h4 className="inline-flex items-center justify-between py-4 text-xl font-medium text-black cursor-pointer lg:text-2xl">
-                    <span>Información General</span>
+                    <span className="colorspan">Información General</span>
```
Comment (Israel-Laguan) [inline src/pages/Product.js]: Please remove extra space at the end

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]
Please rebase and confirm with @Eghinner

File: README.md (hunk 1 above)
Comment (heliomar-pena) [reply README.md:125]: Yes, please rebase

Comment (e-azocar) [reply README.md:125]: Yes, the branch is old, maybe is using the branch "master" or another branch

**Review by e-azocar — APPROVED** [review]

**Pull request closed without merging**